packages/angular_devkit/build_optimizer/src/build-optimizer/build-optimizer_spec.ts

```diff
@@ -88,7 +88,7 @@ describe('build-optimizer', () => {
       });
     });
 
-    it('doesn\'t process files without decorators/ctorParameters/outside Angular', () => {
+    xit('doesn\'t process files without decorators/ctorParameters/outside Angular', () => {
```

Comment:
Why is this test removed?

Reply:
The test should have been failing as `prefix-classes` will add the annotation but `testPrefixClasses` doesn't accept `var Clazz = (function () { function Clazz() { } return Clazz; }());`.

Reply:
I see, this test isn't true anymore at all since we do process all files in some capacity... Maybe it's better to wholly remove the test instead of just doing `xit` then. What do you think?

It's probably true that processing all files is what's leading to slow builds though.

Reply:
Removed the test as it is outdated now.